internal/agentseed/agentseed.go

```diff
@@ -0,0 +1,135 @@
+package agentseed
```

Comment:
Is there a way to make this work for otelcol.exporter.* components?

Reply:
Unclear at present. I'd assume there's some mechanism for including metadata, even though it may not be over http. Needs more research.